module/zfs/zfs_ioctl.c

```diff
@@ -284,6 +285,65 @@ static int zfs_fill_zplprops_root(uint64_t, nvlist_t *, nvlist_t *,
 int zfs_set_prop_nvlist(const char *, zprop_source_t, nvlist_t *, nvlist_t *);
 static int get_nvlist(uint64_t nvl, uint64_t size, int iflag, nvlist_t **nvp);
 
+/* Returns 1 on failure; Returns 0 on success */
```

Comment:
Changes in the shared code (and we probably should extend that to all code source) would need to follow `cstyle -cpP`.

Reply:
The non-Direct interface code changes are removed from this PR and would be added in a separate PR.